test/Ocelot.UnitTests/LoadBalancer/RoundRobinTests.cs

```diff
@@ -39,17 +39,17 @@ public void should_get_next_address()
         }
 
         [Fact]
-        public void should_go_back_to_first_address_after_finished_last()
+        public async Task should_go_back_to_first_address_after_finished_last()
```

Comment:
Not sure that all these async improvements should be a part of PR... Could you cherry pick them to your current PRs please?

Reply:
Latest .NET version is complaining about using GetAwaiter() etc.

Reply:
errors or warnings?

Reply:
OK got it! Not an issue!
It is better to make all signatures `async`.

Reply:
Warnings, but I thought, well it makes sense, why would we use void instead of async Task?

Reply:
BDDfy runs both sync and async methods without problems.